docs.go

```diff
@@ -146,3 +147,35 @@ func flagDetails(flag DocGenerationFlag) string {
 	}
 	return ": " + description
 }
+
+func prepareUsageText(command *Command) string {
+	usageText := ""
+	if command.UsageText != "" {
```

Comment:
Here we've got the entire body of a function inside a single `if` statement. If we flip the conditional with an early return, we'd be able to outdent the business logic and get the complexity down:

```go
func prepareUsageText(command *Command) string {
  if command.UsageText == "" {
    return ""
  }

  // Logic is now at the outermost level
  preparedUsageText := strings.TrimSuffix(command.UsageText, "\n")
  // ...

  return usageText
}